content/en/blog/_posts/2021-12-16-kubernetes-is-moving-on-from-dockershim.md

```diff
@@ -0,0 +1,96 @@
+and projects in the Kubernetes ecosystem.
+
+There are a few container runtimes today that are often lighter, faster and more
+secure than when Docker was the only choice. By removing the dockershim CRI,
+we're embracing the first value of CNCF: "[Fast is better than slow](https://github.com/cncf/foundation/blob/master/charter.md#:~:text=Fast%20is%20better%20than%20slow)".
+Migrating to another runtime will ensure that applications are more portable,
+with fewer dependencies on the infrastructure. Your apps will be more secure,
+with new features unlocked. A different runtime will also let us develop
+features like cgroupv2 support faster.
+
+## Deprecation timeline
+
+We announced dockershim deprecation in December 2020.  Full removal is targeted
+to Kubernetes 1.24, which is planned for April 2022. This deprecation timeline
+aligns with our [deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior),
+which states that deprecated behaviors must function for no less than 1 year
+after their announced deprecation. We'll support version 1.23, which has
+dockershim, for another year in the Kubernetes project (and likely longer with
+managed Kubernetes providers). You'll have time to migrate. If you have more
+questions about the dockershim removal, refer to the [Dockershim Deprecation FAQ](https://k8s.io/dockershim).
+
+We asked you whether you feel prepared for the migration from dockershim in this
+survey: [Are you ready for Dockershim removal](https://kubernetes.io/blog/2021/11/12/are-you-ready-for-dockershim-removal/).
+We had about 600 responses. To everybody who took time filling out the survey,
+thank you.
+
+The results show that we still have a lot of ground to cover to help you to
+migrate smoothly. Several newer runtimes exist, and have been promoted
+extensively. However, a significant number of users still rely on dockershim,
+and sometimes have dependencies that need to be re-worked. Some of these
+dependencies are outside of your control. Based on the feedback received from
+you, here are some of the steps we are taking to help.
+
+## Our next steps
+
+Based on the feedback you provided, we'll be doing the following:
+
+- CNCF will sponsor work on updating documentation on how to migrate from
+  dockershim. We'll use survey feedback and our own experiences migrating from
+  dockershim to, try and cover as many use cases and known issues as possible.
+- We have begun, and will continue to, reach out to various projects and
+  vendors to make sure there is a path for customers migration.
+- The release team for Kubernetes 1.24 has set a high bar for documentation
+  readiness.
+
+If you're part of a project with dependencies on dockershim, or if you're
+interested in helping with the migration effort, please join us! There's always
+room for more contributors, whether to our transition tools or to our
+documentation. To get started, say hello in
```

Comment:
```suggestion
If you want to see the changes for yourself, pull down an alpha release here: LINK TBD. 

If you want to get involved, the documentation team would love your help! Drop into [#sig-docs-dockershim](https://kubernetes.slack.com/archives/C02PZMA3R6E) where you can help migrate existing code samples to containerd, update diagrams, and more.
```

Reply:
not sure what changes? Do you mean 1.24 beta release?

Reply:
Maybe let's not signpost people to that Slack channel at all? If we wanted to, I'd put an article up on https:/k8s.dev/blog/ instead.